examples/calculator_fractional/src/main.rs

```diff
@@ -121,7 +121,7 @@ fn encrypt_term(runtime: &FheRuntime, public_key: &PublicKey, input: Term) -> Te
         Term::Ans => Term::Ans,
         Term::F64(v) => Term::Encrypted(
             runtime
-                .encrypt(Fractional::<64>::try_from(v).unwrap(), public_key)
+                .encrypt(Fractional::<64>::from(v), public_key)
```

Comment:
Why is this allowed? What if the value to pass for conversion is NaN, Infinity out of range?

Reply:
All of these were just clippy fixes. The warning is basically telling you when there's a `From` instance, just use it, rather than introduce the noise of trying something that is infallible. The existing code works because `T` impls `TryFrom<U>` where `T: From<U>`, with an infallible error type.

So, this code is semantically the same, just less noise.

> What if the value to pass for conversion is NaN, Infinity out of range?

If we want to handle these, then we should implement `TryFrom<f64>` rather than [`From<f64>` for `Fractional`](https://github.com/Sunscreen-tech/Sunscreen/blob/5cccd88ac0ee4ab53287a4ee0a0fed2126abd2b4/sunscreen/src/types/bfv/fractional.rs#L560-L564).